File: example/demo/recipes/remove_double_negation/tests/java/input/in_expression.java
class Test {
    Test() {
        //print(!(! true))
        print(/*comment1*/!/*comment2*/(!/*comment3*/true/*comment4*/)/*comment5*/);
        //print(!(! true && false))
        print(!(!true && false));
        //print(!(! true || false))
        print(!(!true || false));
        //print(! ! true)
        print(!!true);
        //print(! ! false)
        print(!!false);
        //print(! ! ! true)
        print(!!! true);
        //print(! ! ! ! true)
        print(!!!!true);
        //print(!(((! ! ! true))))
        print(!(((!!!true))));

        //print(!(!! true || false))
        print(!(!!true || false));

        //if (!(! testWhatever(12))) return "Hello"
        if (!(!testWhatever(12))) return "Hello";
    }
}

File: example/demo/recipes/remove_double_negation/tests/python/expected/in_expression.py
# print(not (not True))
print(True)
# print(not (not True and False))
print(not (not True and False))
# print(not not True)
print(True)
# print(not not False)
print(False)
# print(not not not True)
print(not True)
# print(not not not not True)
print(True)
# print(not (((not not not True))))
print(True)
# print(not (not not True and False))
print(not (True and False))


# if (not (not testWhatever(12))):
#     return "Hello"
if (testWhatever(12)):
    return "Hello"
